python/tk_nuke_writenode/handler.py

```diff
@@ -87,7 +90,7 @@ def populate_profiles_from_settings(self):
         for profile in self._app.get_setting("write_nodes", []):
             name = profile["name"]
             if name in self._profiles:
-                self._app.log_warning("Configuration contains multiple Write Node profiles called '%s'!  Only the "
+                self._app.engine.log_warning("Configuration contains multiple Write Node profiles called '%s'!  Only the "
```

Comment:
The app logger should work fine. Have you debugged this any further?

Reply:
Ah, I was focusing on getting the gizmo to work and left in these changes....
Opened http://shotgun.d2-india.com/detail/Ticket/1840. Merging this for now.

Reply:
Since I didn't have permission to merge anyway, reverted these changes. And pushed additional fixes to the gizmo.